akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -300,6 +300,7 @@ private[stream] object VirtualProcessor {
   case object Completed extends Termination
   case class Failed(ex: Throwable) extends Termination
 
+  // FIXME this is not a legal Subscriber in the RS sense as a Subscriber may never be attached to more than 1 Publisher
```

Comment:
Should I wait for #17831 to merge?
https://github.com/akka/akka/blob/wip-flow-fold-%E2%88%9A/akka-stream/src/main/scala/akka/stream/impl/CompletedPublishers.scala#L82